### PR TITLE
Implement Laravel coding standards

### PR DESCRIPTION
**Pull request by ArjenMiedema**
The coding standards for Laravel are connected now by requiring
a new package for Laravel specific rulesets.

### CODE REVIEW

**Comment by leonhelmus** [comment]
Only change bitbucket file to php 74 and i will merge it.

**Comment by ArjenMiedema** [comment]
@leonhelmus updated the Bitbucket file

**Review by leonhelmus — APPROVED** [review]

**Comment by leonhelmus** [comment]
Could you also add laravel to the mapping: https://github.com/mediact/testing-suite/commit/7ac0b41bd9fdd92387d72081948640bc8a05dcdf

File: templates/files/laravel/bitbucket-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+image: srcoder/development-php:php72-fpm
```
[the remark this answers is not in the document]
Comment (ArjenMiedema) [reply templates/files/laravel/bitbucket-pipelines.yml]: Yeah, I don't use those files anymore, so I will most likely forget this for every pull request :-P

**Pull request merged**